Bindings/Java/Matlab/tests/testC3Dconverter.m

```diff
@@ -0,0 +1,138 @@
+% ----------------------------------------------------------------------- %
+% The OpenSim API is a toolkit for musculoskeletal modeling and           %
+% simulation. See http://opensim.stanford.edu and the NOTICE file         %
+% for more information. OpenSim is developed at Stanford University       %
+% and supported by the US National Institutes of Health (U54 GM072970,    %
+% R24 HD065690) and by DARPA through the Warrior Web program.             %
+%                                                                         %
+% Copyright (c) 2005-2017 Stanford University and the Authors             %
+% Author(s): James Dunne                                                  %
+%                                                                         %
+% Licensed under the Apache License, Version 2.0 (the "License");         %
+% you may not use this file except in compliance with the License.        %
+% You may obtain a copy of the License at                                 %
+% http://www.apache.org/licenses/LICENSE-2.0.                             %
+%                                                                         %
+% Unless required by applicable law or agreed to in writing, software     %
+% distributed under the License is distributed on an "AS IS" BASIS,       %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+% implied. See the License for the specific language governing            %
+% permissions and limitations under the License.                          %
+% ----------------------------------------------------------------------- %
+
+% Function tests osimC3Dconverter for correct writing of trc and mot files,
+% and that rotations are done correctly.
+
+% Author: James Dunne
+
+%% Load libraries
+import org.opensim.modeling.*
+
+%% get path to file
+filepath = 'walking2.c3d';
+[path, file, ext] = fileparts(filepath);
+if isempty(exist('walking2.c3d', 'file'))
+   error('cannot find test file (walking2.c3d) at walking2.c3d');
+end
+
+%% Instantiate the file readers
+trc = TRCFileAdapter();
+sto = STOFileAdapter();
+
+%% Test script for utility function
+osimC3Dconverter('filepath', filepath);
+
+%% Test if trc and mot file were printed.
+assert( exist(fullfile(path,[file '.trc']), 'file') == 2, 'TRC was not printed ')
+assert( exist(fullfile(path,[file '.mot']), 'file') == 2, 'TRC was not printed ')
+
+%% Test rotations
+% test if roations about X, Y, and Z, respectively, maintain the correct
+% components. ie if rotatiing about X, X values remain the same.
```

Comment:
Right, `~=` will require that it changed, that's good! But it can still be wrong and still pass if it is changed at all? E.g. rotated 90 instead of -90 would pass, right?